test/test_examples.py

```diff
@@ -1137,6 +1137,42 @@ def test_versioned_api(self):
             uri, server_api=ServerApi("1", deprecation_errors=True))
         # End Versioned API Example 4
 
+    @client_context.require_version_min(4, 7)
```

Comment:
Do we still need to skip sharded clusters?

Reply:
The issue has been fixed in 5.0.2 - https://jira.mongodb.org/browse/SERVER-58794 but seems like we are still using 5.0. 🤔 
Should I just skip it on sharded or we can wait for this to pass before merging?

Reply:
Do you mind updating drivers evergreen tools to use 5.0.2 real quick?

Reply:
How does one figure out if it is available yet? I only see 5.0.1 on `m`...

Reply:
Yeah its definitely not yet available.

Reply:
It's not out yet: https://docs.mongodb.com/manual/release-notes/5.0/

I suggest putting something like :
```
if sharded and 5.0 and <5.0.2:
    self.skip()
```